### PR TITLE
jalde krde

### PR DESCRIPTION
**Pull request by SarthakSahu1009**

### CODE REVIEW

**Comment by ridhishjain-zepto** [comment]
@SarthakSahu1009 what are u trying to achieve?

**Comment by ridhishjain-zepto** [comment]
@VasuSehgal123 did the same

File: 91ABeta.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#include <bits/stdc++.h>
```
Comment (rishabh-997) [inline 91ABeta.cpp:1]: Add a license

**Comment by KomoderHell** [comment]
kindly explain in detail?

File: 91ABeta.cpp (hunk 1 above)
Comment (ridhishjain-zepto) [reply 91ABeta.cpp:1]: Yes indeed, it's not at all official

File: 91ABeta.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+using namespace std;
```
Comment (rishabh-997) [inline 91ABeta.cpp:3]: don't use this, create your own namespace

File: 91ABeta.cpp (hunk 1 above)
Comment (KomoderHell) [reply 91ABeta.cpp:1]: also test cases

File: 91ABeta.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+int main()
+{
+
```
Comment (rishabh-997) [inline 91ABeta.cpp:7]: remove extra line

File: 91ABeta.cpp [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    int n;
+    cin>>n;
```
Comment (rishabh-997) [inline 91ABeta.cpp:9]: align it properly

File: 91ABeta.cpp [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    bool ch = true;
```
Comment (rishabh-997) [inline 91ABeta.cpp:10]: ch??? chutiya likhe ho kya

**Comment by ridhishjain-zepto** [comment]
One hell of a review

File: 91ABeta.cpp [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+    while(n!=0)
```
Comment (rishabh-997) [inline 91ABeta.cpp:12]: matlab maaa chod di indent ki

File: 91ABeta.cpp (hunk 5 above)
Comment (ridhishjain) [reply 91ABeta.cpp:10]: :smile: :

File: 91ABeta.cpp [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        int temp = n%10;
+        n = n/10;
+        if((temp != 4) || (temp!=7))
+        {
+            ch = false;
+            break;
+        }
+
```
Comment (rishabh-997) [inline 91ABeta.cpp:21]: remove extra line

File: 91ABeta.cpp [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    if(ch)
+    {
+        cout<< "YES";
+    }else{
```
Comment (KomoderHell) [inline 91ABeta.cpp:26]: use tabs not spaces

File: 91ABeta.cpp [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+        n = n/10;
+        if((temp != 4) || (temp!=7))
+        {
+            ch = false;
+            break;
+        }
+
+    }
+    if(ch)
+    {
```
Comment (rishabh-997) [inline 91ABeta.cpp:24]: make it if(ch) {
...
}

File: 91ABeta.cpp (hunk 8 above)
Comment (rishabh-997) [inline 91ABeta.cpp:26]: aisa kar, ek hi line mein likh de poora... penchodaaaa

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Change kar bsdka

File: 91ABeta.cpp (hunk 5 above)
Comment (ridhishjain-zepto) [reply 91ABeta.cpp:10]: abey sale

**Comment by rishabh-997** [comment]
> Change kar bsdka

isko acknowledge nahi karoge, main merge nahi karega

**Comment by KomoderHell** [comment]
@SarthakSahu1009 address the changes

**Comment by ridhishjain-zepto** [comment]
@SarthakSahu1009 is toyed with

**Pull request merged**